src/custom-blocks/auto-item-list/index.js

```diff
@@ -0,0 +1,468 @@
+import { store as coreStore } from '@wordpress/core-data';
+
+const { __ } = wp.i18n;
+const { registerBlockType, registerBlockStyle } = wp.blocks;
+const { Fragment } = wp.element;
+const { useSelect } = wp.data;
+const { InspectorControls, MediaUpload, InnerBlocks } = wp.blockEditor;
+const { PanelBody, SelectControl, Button, TextControl, ToggleControl, RadioControl} = wp.components;
+const allowedMediaTypes = ['image'];
+
+import { __experimentalNumberControl as NumberControl } from '@wordpress/components';
+import { __experimentalText as Text } from '@wordpress/components';
+
+registerBlockType("mojblocks/auto-item-list", {
+  title: __("Most Recent Items", "mojblocks"),
```

Comment:
Not sure this is the right name considering  it can be more then most recent if agreed with DM and PM for MVP then should not stop deploy

Reply:
Changed to `Items List`